cargo/flatpak-cargo-generator.py

```diff
@@ -74,6 +74,27 @@ async def get_remote_sha256(url):
     return sha256.hexdigest()
 
 
+def find_toml(git_repo_dir):
+    if os.path.isfile(os.path.join(git_repo_dir, 'Cargo.toml')):
+        toml_data = load_toml(os.path.join(git_repo_dir, 'Cargo.toml'))
+        toml_dir = '.'
+    else:
+        toml_list = []
+        for entry in os.scandir(git_repo_dir):
+            if entry.is_dir():
+                toml_path = os.path.join(git_repo_dir, entry.name, 'Cargo.toml')
+                if os.path.isfile(toml_path):
+                    toml_list.append((toml_path, entry.name))
+        if len(toml_list) == 1:
+            toml_data = load_toml(toml_list[0][0])
+            toml_dir = toml_list[0][1]
+        elif len(toml_list) > 1:
+            raise Exception(f'Multiple Cargo.toml files found in {git_repo_dir}')
+        else:
+            raise Exception(f'No Cargo.toml found in {git_repo_dir}')
```

Comment:
We potentially can handle repos with multiple `Cargo.toml`s in subdirectories and without top-level `Cargo.toml`, just not yet. Maybe raise a `NotImplementedError` a suggest the user to file an issue pointing to the the unsupported repo?